### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by AbednegoKibitok**
#Hello Microverse Project
-Create index.html file.
-Create styles.css file.
-Create test.md file
-Install linters
-Create Package. json file
-Create Package-log. json file
-Edit README.md
-Update .gitignore file
-Update README.md
-Run a git rm test.md as requested by code reviewer.

### CODE REVIEW

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @AbednegoKibitok , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

 - [ ] Great job in the configuration of linters, but I see that there is an error bug at the stylelint linter, so please fix this linter by adding this command at the terminal
` npx stylelint "**/*.{css,scss}" --fix `,so the linter will be happy for that smiley :+1: 

- [ ]  You did well on the project, but I see that you didn't write a pull request for this project, so please add a Pull Request which should have a descriptive title telling you what changes are inside and Pull Request should have a summary describing introduced changes in more detail :+1: 

eg.
image

- [x] Great job :+1:  adding the node modules to the gitignore file, but I see that you didn't put the  test.md file inside the gitignore, so kindly add the  test.md file inside the gitignore as it also part of the requirement
 
- [x]  kindly Add a descriptive README  to your project - please use [this template](https://github.com/microverseinc/readme-template) as required in the project but customize it to your project and add also the license it will give a small description and documentation to your project. :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [x] [OPTIONAL]  Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Add or Upload all files " instead of "All Files uploaded]" ... etc +1 :heavy_check_mark:👨🏻‍💻
 - [x] [OPTIONAL] Also consider cleaning the gitignore file from all the unnecessary things just keep the node_modules and the test.md inside it :+1: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by AbednegoKibitok** [comment]
Hello Microverse! project

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+This should be ignored by git
+test.md
 # Logs
 logs
 *.log
```
Comment (franclobo) [inline .gitignore]: - [ ] As the last code reviewer said: 

> Great job 👍 adding the node modules to the gitignore file, but I see that you didn't put the test.md file inside the gitignore, so kindly add the test.md file inside the gitignore as it also part of the requirement.

- [ ] Please 🙏🏽 add `./test.md` to the `.gitigore` file, to ignore the file by git,  when you correctly added the file to the `.gitignore` the test.md file doesn't charge in your repository:

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @AbednegoKibitok, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As the last code reviewer said kindly complete the readme following the README template by Microverse. All main headers from the 👉🏽 [README template](https://github.com/microverseinc/readme-template) should be present in your repo for professional documentation.
- [ ] As the last code reviewer said, please 🙏🏽 add a descriptive PR that contains a title and a resume of the actions for professional documentation.

May I help you? 👇🏽 

https://user-images.githubusercontent.com/58642949/212052119-17d612f5-0a68-43ba-83b6-994cb6518e63.mp4

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (joskalenda) [inline test.md]: - [ ] When a file is ignored by GitHub (`added in the .gitignore file`), it should not be present on the repository either, you have done fine as said by the previous reviewers but the issue is your test.md file is still visible on the repository, consider running this command `git rm test.md` to remove it from the repo and you will be good to go afterward.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @AbednegoKibitok , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Good pull request summary
- [ ].. ✅ Updated readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @AbednegoKibitok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**